src/utils.ts

```diff
@@ -37,15 +37,14 @@ export function parseNumber(s: string): number | undefined {
 }
 
 export function parseParameters(parameterOverrides: string): Parameter[] {
-  try {
-    const path = new URL(parameterOverrides)
```

Comment:
Removing this changes the behavior. It uses `URL` to support using parameters to fetch from `http` or `https`.